packages/node/src/methods/app-instance/propose-install/controller.ts

```diff
@@ -53,6 +47,12 @@ export default class ProposeInstallController extends NodeController {
     params: Node.ProposeInstallParams
   ) {
     const { store, publicIdentifier, networkContext } = requestHandler;
+    const { initialState } = params;
```

Comment:
This change should not be reverted as we wouldn't want to queue an invalid request.

Reply:
It should be in `beforeExecution`

Reply:
That doesn't make sense though - why would you queue a request that's invalid to begin with?

Reply:
`beforeExecution` runs before the queue is added